# Description: Some Thinkpads have ACPI brightness controls, disable laptop_panel.brightness_in_hardware for them.
--- hal-info-20071212/fdi/information/10freedesktop/10-laptop-panel-hardware.fdi	2007-12-12 17:51:08.000000000 +0100
+++ hal-info-20071212.new/fdi/information/10freedesktop/10-laptop-panel-hardware.fdi	2008-02-21 20:28:23.000000000 +0100
@@ -26,5 +26,8 @@
         </match>
       </match>
     </match>
+    <match key="linux.sysfs_path" contains="/sys/devices/virtual/backlight/acpi_video">
+      <merge key="laptop_panel.brightness_in_hardware" type="bool">false</merge>
+    </match>
   </device>
 </deviceinfo>
